src/Config/Method.php

```diff
@@ -0,0 +1,90 @@
+<?php
+namespace GravApi\Config;
+
+/**
+ * Class Method
+ * @package GravApi\Config
+ */
+class Method
+{
+    /**
+     * @var boolean
+     */
+    protected $enabled = false;
+
+    /**
+     * @var boolean
+     */
+    protected $useAuth = false;
+
+    /**
+     * @var array
+     */
+    protected $fields = array();
+
+    /**
+     * @var array
+     */
+    protected $ignore_files = array();
+
+    public function __construct(array $config = array())
+    {
+        if (isset($config['enabled'])) {
+            $this->enabled = $config['enabled'];
+        }
+
+        if (isset($config['auth'])) {
+            $this->auth = $config['auth'];
+        }
+
+        if (isset($config['fields'])) {
+            // TODO: field string array validation
+            $this->fields = $config['fields'];
```

Comment:
Can use the `ArrayHelper` class to parse the `$config['fields']` array and only return items which are strings, dropping non-string values.

Reply:
This also applies to `ignore_fields`.